README.md

```diff
@@ -21,7 +21,7 @@ Provide nice configuration layer, secured capture controller, storages integrati
 ## Resources
 
 * [Site](https://payum.forma-pro.com/)
-* [Documentation](https://github.com/Payum/Payum/blob/master/docs/index.md#symfony-payum-bundle)
+* [Documentation](https://github.com/Payum/Payum/blob/master/docs/symfony/get-it-started.md)
```

Comment:
Thanks @sovetski. Can you maybe update it to the following URL:

`https://payum.gitbook.io/payum/symfony/get-it-started`

This is the new docs home